components/atoms/figure-component.html

```diff
@@ -0,0 +1,59 @@
+<div class="component" title="figure Component" version="1.0">
+<style>
+.figure {
+    display: grid;
+    margin: 0 auto;
+}
+
+.figure img {
+    width: 100%;
+    margin-bottom: 1em;
+}
+
+.figure figcaption {
+    text-align: center;
+}
+.figure figcaption a{
+    text-decoration: none;
+}
+</style>
+
+
+  <figure class="figure">
+      <img src="../../img/mustafa.jpg" alt="">
+    <figcaption>
+        <span>Mustafa Alsaeedi</span>
+        <a href="http://MustafaAlsaeedi.tk/">Website: MustafaAlsaeedi.tk</a>
+        <a href="https://github.com/MustafaAlsaeedi">GitHub Profile</a>
+        <a href="https://www.linkedin.com/in/mustafa-a-b207a4150/">LinkedIn Profile</a>
+    </figcaption>
+  </figure>
+
+  <figure class="figure">
```

Comment:
No need to have more than one example in the component unless you are trying to show an example that looks very different.